openassessment/xblock/student_training_mixin.py

```diff
@@ -50,7 +50,7 @@ def render_student_training(self, data, suffix=''):   # pylint:disable=W0613
         try:
             path, context = self.training_path_and_context()
         except: # pylint:disable=W0702
-            msg = u"Could not render student training step for submission {}".format(self.submission_uuid)
+            msg = u"Could not render learner training step for submission {}".format(self.submission_uuid)
```

Comment:
Nit: there is an inconsistency as to the capitalization of "Learner Training step". Most other usages in this PR go with the capitalization.